core/src/test/java/com/linecorp/armeria/common/logging/DefaultRequestLogTest.java

```diff
@@ -367,6 +368,46 @@ void logServiceNameWithServiceNaming_of() {
         assertThat(log.serviceName()).isEqualTo("hardCodedServiceName");
     }
 
+    @Test
+    void logServiceNameWithServiceNaming_shorten50() {
```

Comment:
Could you fork the upstream test suites for TargetLengthBasedClassNameAbbreviator.
That might be useful when we optimize and refactor TargetLengthBasedClassNameAbbreviator.
https://github.com/qos-ch/logback/blob/c2dcbfcfb4048d11d7e81cd9220efbaaccf931fa/logback-classic/src/test/java/ch/qos/logback/classic/pattern/TargetLengthBasedClassNameAbbreviatorTest.java

Reply:
Sure. FYI, `c2dcbf` is latest commit. There is no upstream.